howso/update_cases.amlg

```diff
@@ -127,42 +127,42 @@
 	;returns null if invalid features specified or an assoc with "count"
-	;
-	;parameters:
-	; features: list of names of feature to edit
-	; feature_values: list of values corresponding to features
-	; case_indices: optional, list of pair (list) of session id and index, where index is the original 0-based session_training_index of the case as
-	; 		it was trained. If specified, ignores condition and condition_session
-	; condition_session: optional, if specified ignores condition and operates on cases for the specified session id
-	; condition: assoc of feature->value(s)
-	;		no value = must have feature
-	;   	- for continuous or numeric ordinal features:
-	;			one value = must equal exactly the value or be close to it for fuzzy match
-	;			two values = inclusive between
-	;   	- for nominal or string ordinal features:
-	;			n values = must match any of these values exactly
-	; num_cases: optional, limit on the number of cases to edit; If set to zero there will be no limit.
-	;		If null, will be set to k if precision is "similar" or no limit if precision is "exact". default is null
-	; precision: optional string,  default is 'exact', used only with 'condition' parameter, will find exact matches if 'exact' and similar cases if 'similar'.
-	; session: the session id when this call !is being made
 	#edit_cases
 	(declare
 		(assoc
 			;{type "list" values "string"}
+			;list of names of feature to edit
+			; feature_values: 
 			features (list)
 			;{type "list"}
+			;list of values corresponding to features
 			feature_values (list)
 			;{ref "CaseIndices"}
+			;optional, list of pair (list) of session id and index, where index is the original 0-based session_training_index of the case as
+			;	it was trained. If specified, ignores condition and condition_session
 			case_indices (list)
 			;{ref "Condition"}
+			;assoc of feature->value(s)
+			;		no value = must have feature
+			;   	- for continuous or numeric ordinal features:
+			;			one value = must equal exactly the value or be close to it for fuzzy match
+			;			two values = inclusive between
+			;   	- for nominal or string ordinal features:
+			;			n values = must match any of these values exactly
 			condition (assoc)
 			;{type "string"}
+			;optional, if specified ignores condition and operates on cases for the specified session id
 			condition_session (null)
 			;{type "number" min 0}
+			;optional, limit on the number of cases to edit; If set to zero there will be no limit.
+			;	If null, will be set to k if precision is "similar" or no limit if precision is "exact". default is null
 			num_cases (null)
 			;{type "string" enum ["exact" "similar"]}
+			;optional string,  default is 'exact', used only with 'condition' parameter, will find exact matches if 'exact' and similar cases if 'similar'.
+			; session: 
 			precision "exact"
 			;{type "string"}
+			;the session id when this call !is being made
```

Comment:
dubious exclamation point

Reply:
Found multiple locations with this error too, all addressed in d8a9404f922319cfd8e1c60555d514603ff45ec3